bancor_research/scenario_generator/monte_carlo_generator.py

```diff
@@ -0,0 +1,616 @@
+from decimal import Decimal
```

Comment:
You can import it directly `from bancor_research` (as done elsewhere in the code), where the `Decimal` class is also properly configured